client/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@hirosystems/token-metadata-api-client",
+  "version": "2.0.0",
+  "description": "Client for @hirosystems/token-metadata-api",
+  "author": "Hiro Systems PBC <engineering@hiro.so> (https://hiro.so)",
+  "keywords": [
+    "fetch",
+    "typescript",
+    "swagger-client",
+    "@hirosystems/token-metadata-api-client"
+  ],
+  "license": "GPL-3.0",
+  "main": "./lib/index.js",
+  "typings": "./lib/index.d.ts",
+  "unpkg": "lib/index.umd.js",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "build": "rimraf ./lib && npm run build:client --prefix .. && npm run build:node && npm run build:browser",
+    "build:node": "tsc && cp -R src/generated lib/generated",
```

Comment:
@zone117x I noticed you didn't copy the generated types to the `lib` dir in the Stacks API client. Not sure why that never worked for me in my tests so I had to do this instead. I also tried including the `src` directory in the `"files"` entry in this file but it made no difference because the included path was still incorrect.

Let me know if you have other ideas.

Reply:
We actually had to do the same thing here: https://github.com/hirosystems/stacks-blockchain-api/pull/2110

I think you can also have the generator output a `.ts` file rather than a `.d.ts` file and then typescript will include a final `.d.ts` in the output dir.